### ISSUE TITLE
JSONEditor library not loaded in form assets

### ISSUE THREAD

**Issue by bendavis78** [jmrivas86/django-json-widget#55]
* django-json-widget version: master
* Django version: 2.2.12
* Python version: 3.x

When using the JSONEditor widget on a basic modelform (non-admin page), the JSONEditor library is not loaded, resulting in the following javascript error:

    Uncaught ReferenceError: JSONEditor is not defined

**Comment by AnderUstarroz** [comment]
yes, same problem here, current version seems to be broken.

**Comment by AnderUstarroz** [comment]
Actually.. seems like it could be fixed just by adding this within your Django settings.py:

```
JSON_EDITOR_CSS = 'dist/jsoneditor.min.css'
JSON_EDITOR_JS = 'dist/jsoneditor.min.js'
```

**Comment by kevindice** [comment]
Yup - same here after bumping 1.0.1 to 1.10.

Looks like #53 and #52 both fix this.

**Comment by StevenMapes** [comment]
Yeah this is fixed in v1.1.1 - https://github.com/jmrivas86/django-json-widget/releases/tag/v1.1.1

**Comment by kevindice** [comment]
Thanks!

**Comment by jmrivas86** [comment]
Fixed in version 1.1.1 by @StevenMapes 👏  in https://github.com/jmrivas86/django-json-widget/pull/52
